packages/mocks/package.json

```diff
@@ -4,23 +4,17 @@
   "private": true,
   "repository": "git@github.com:kubev2v/forklift-console-plugin.git",
   "license": "Apache-2.0",
-  "main": "./src/cluster-proxy-server/server.ts",
+  "main": "./src/index.ts",
   "scripts": {
     "postinstall": "npx msw init ./generated --save",
```

Comment:
i think we don't need the `postinstall` now ?

Reply:
@sjd78 @yaacov 
Right now PR #392 imports service runner from `mocks` package. This should be changed from there if needed.